src/Poseidon/SequencingSource.hs

```diff
@@ -273,13 +396,17 @@ readSeqSourceFile seqSourcePath = do
     -- error case management
```

Comment:
Just a few lines up, in L 391: Cleaning out quotation marks from header names is fine, but technically might break things if people (bizarrely) would put escaped quotes into header strings right? I think we can accept that.

Reply:
I copied this from the .janno code. Might indeed have weird effects in edge cases. I added a comment now to warn of the danger.